Plugins/BTCPayServer.RockstarDev.Plugins.TransactionCounter/CounterPluginSettings.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using BTCPayServer.RockstarDev.Plugins.TransactionCounter.ViewModels;
+
+namespace BTCPayServer.RockstarDev.Plugins.TransactionCounter;
+
+public class CounterPluginSettings
+{
+    public DateTime StartDate { get; set; }
+    public DateTime? EndDate { get; set; }
+    public bool Enabled { get; set; }
+    public bool AllStores { get; set; }
+    public string? Password { get; set; }
+    public string? CustomHtmlTemplate { get; set; }
+    public string? BackgroundVideoUrl { get; set; }
```

Comment:
Remove `BackgroundVideoUrl` variable, and rename `CustomHtmlTemplate` to `HtmlTemplate`.

Then populate `HtmlTemplate` with the default one on initialization and display it for editing. When loading counter HTML page we will just be replacing {COUNTER} value. How it should look:

![image](https://github.com/user-attachments/assets/ff1003ad-2410-4ee2-a670-e09c79665909)